common/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -331,6 +331,10 @@ export class ActiveKeyBase {
       }
     }
 
+    if(!key.text && typeof key.id == 'string') {
+      key.text = ActiveKey.unicodeIDToText(key.id);
+    }
+
```

Comment:
Reference:

https://github.com/keymanapp/keyman/blob/dc368c9168a2b53442b813d035304d3486347209/common/web/keyboard-processor/src/keyboards/activeLayout.ts#L217-L221

If it's not a `U_`  key, no text-adjustment will occur here.